### PR TITLE
gajim: update to 1.9.0, adopt.

### PR DESCRIPTION
**Pull request by Emru1**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, amd64-glibc

Python3-nbxmpp is here too, as dependency

### CODE REVIEW

**Comment by AnInternetTroll** [comment]
Tested on x86_64-glibc with both a public room (through biboumi) and an
OMEMO encrypted room. Works as expected!

**Comment by lissine0** [comment]
Hi, it also works for me on x86_64-glibc

A problem I have (which exists before this update), is that by default, the emojis are displayed in black and white, and many emojis sent by other users can't be decyphered.
(I have the freetype package installed.)

default
default

Installing font-awesome6 gave me more black and white emojis
And installing noto-fonts-emoji or  font-emoji-one-color gave me a lot of colored emojis

font-emoji-one-color:
font-emoji-one-color_1
font-emoji-one-color_2

noto-fonts-emoji: (which is the emoji font shown in the official screenshots https://gajim.org/img/news/message-reactions.png )
noto-fonts-emoji_1
noto-fonts-emoji_2


What do you think? should we add noto-fonts-emoji as a dependency? (I think it looks better than font-emoji-one-color)

Or is this issue only in my setup? (I have emojis working correctly on Firefox)

**Comment by elliniyessine** [comment]
It turns out font-emoji-one-color is a proprietary font, and we may not use it for free for non-profit use https://joypixels.com/licenses

I suggest Twitter's open source twemoji (shown here)
twemoji

Or we can use Google's open source noto-fonts-emoji (shown in the previous comment)
twemoji is smaller to download (4MB vs 18MB) and uses less disk space (14MB vs 19MB) 
(Plus I think twemoji looks better)

**Comment by Emru1** [comment]
I think that we shouldn't use any emoji pack as dependency. User shouldn't be forced to use/download any packages that aren't strictly required.
We could add information to `README.voidlinux` suggesting some emoji packages.

**Comment by Emru1** [comment]
@lissine0 @elliniyessine I added emoji packages to README
I also symlinked README to INSTALL.msg

File: srcpkgs/gajim/INSTALL.msg
Comment (classabbyamp) [inline srcpkgs/gajim/INSTALL.msg:1]: don't use INSTALL.msg for this

Comment (Emru1) [reply srcpkgs/gajim/INSTALL.msg:1]: removed

**Pull request merged**